src/lib/buyables/tokkulBuyables.ts

```diff
@@ -0,0 +1,226 @@
+import itemID from '../util/itemID';
+
+interface TokkulShop {
+	name: string;
+	inputItem: number;
+	tokkulReturn: number;
+	tokkulCost?: number; // Prices are using the Karamja gloves 1
```

Comment:
dont put a comment at the end of the line.
It should be at the top of the like, like this:
```
/**
* Prices are using the Karamja gloves 1
*/
tokkulCost?: number;
```

However, I dont think i want to use karamja gloves prices, considering we might add achievement diaries at some point.